javascript-bank/bank.js

```diff
@@ -0,0 +1,51 @@
+/* exported Bank */
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (Number.isInteger(balance) && balance > 0) {
+    var account = new Account(this.nextAccountNumber, holder);
+    account.deposit(balance);
+    this.accounts.push(account);
+    this.nextAccountNumber++;
+    return account.number;
+  }
+  return null;
+};
+
+Bank.prototype.getAccount = function (number) {
+  for (var i = 0; i < this.accounts.length; i++) {
+    if (this.accounts[i].number === number) {
+      return this.accounts[i];
+    }
+  }
+  return null;
+};
+
+Bank.prototype.getTotalAssets = function () {
+  if (this.accounts.length === 0) {
+    return 0;
+  }
```

Comment:
This can be removed. Since the length is zero, the loop would never start, so you would return balance, which starts at zero. Same result, less code 😎